docs/resources/Channel.md

```diff
@@ -202,6 +202,7 @@ Represents a message sent in a channel within Discord.
 | application?              | [message application](#DOCS_RESOURCES_CHANNEL/message-object-message-application-structure) object                                              | sent with Rich Presence-related chat embeds                                                                                   |
 | message_reference?        | [message reference](#DOCS_RESOURCES_CHANNEL/message-object-message-reference-structure) object                                                  | reference data sent with crossposted messages                                                                                 |
 | flags?                    | integer                                                                                                                                         | [message flags](#DOCS_RESOURCES_CHANNEL/message-object-message-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field) |
+| stickers?                 | array of [sticker](#DOCS_RESOURCES_CHANNEL/message-sticker-structure) objects                                                            | the stickers sent with the message                                                                                            |
```

Comment:
This needs to be `#DOCS_RESOURCE_CHANNEL/message-object-message-sticker-structure`

Reply:
Also might be worth saying in the description that bots can only receive stickers, and not send them